README.md

```diff
@@ -12,10 +12,10 @@ This library takes API Blueprint AST and returns specific HTTP transactions (Req
 - Inherits parameters from parent Resource and Action sections
 - Expands URI templates
 - Warns on undefined URI query and path parameters
-- Validates URI parameteres types
-- Selects first request and first response if multiple request or responses are given in the 
-- Assigns path origin object pointing to the Blueprint AST 
-- Compiles [cannonical transaction path](#canonical-transaction-paths) as a unique identifier for each transaction
+- Validates URI parameters types
+- Selects first request and first response if multiple request or responses are given in the
```

Comment:
This still doesn't make sense.

"if multiple request or responses are given in the..." it's missing some word after the? (I don't know what it is)